core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -917,6 +915,17 @@ public ServerBuilder annotatedService(Object service,
                                                                     "exceptionHandlersAndConverters")));
     }
 
+    /**
+     * Binds the specified annotated service object under the path prefix {@code "/"}.
+     *
+     * @param customizer the {@link Consumer} which customizes the given
+     *                   {@link AnnotatedHttpServiceConfiguratorSetters}
+     */
+    public ServerBuilder annotatedService(Object service,
+                                          Consumer<AnnotatedHttpServiceConfiguratorSetters> customizer) {
```

Comment:
I'm not sure that we need to introduce this `AnnotatedHttpServiceConfiguratorSetters` because we add the setters for handlers to `AnnotatedServiceBindingBuilder`.
Because there're a lot of methods in `ServerBuilder` already, I hope to remove this and other ones that take `exceptionHandlersAndConverters` and just leave the ones that take the `service`, `path` and `decorator`. A user can still set the configuration using `annotatedService()` when he or she wants to set service level configuration as we do `route()` for `ServiceBindingBuilder`.
What do you think? @trustin, @anuraaga and @ikhoon

Reply:
SGTM

Reply:
Agreed. If `AnnotatedHttpServiceConfigurator` exists for `exceptionHandlers`, `requestConverters`, and `responseConverters`, I think AnnotatedServiceBindingBuilder is enough for this case.

Reply:
Thanks for reviews!

I will remove relate to `AnnotatedHttpServiceConfiguratorSetters`.